src/objects/npe01__OppPayment__c.object

```diff
@@ -100,6 +100,18 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Batch_Number__c</fullName>
```

Comment:
@melissabarber have you had a chance to review help text and description for these fields?

Reply:
And should the text indicate in some way that this field is intended for use by automation and shouldn't be re-purposed casually?

Reply:
Really good point about talking about the automation. I added suggestions to the help text and descriptions for both fields to note this, @dfuller114.